frontend/packages/knative-plugin/src/components/overview/DeploymentOverviewList.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { PodControllerOverviewItem } from '@console/shared';
+import { SidebarSectionHeading } from '@console/internal/components/utils';
+import { ResourceLink } from '@console/internal/components/utils';
```

Comment:
Conbine this two imports.
import { ResourceLink, SidebarSectionHeading } from '@console/internal/components/utils';